### ISSUE TITLE
Unable to defocus certain fields in Safari

### ISSUE THREAD

**Issue by quincylvania** [openstreetmap/iD#9842]
### How to reproduce the issue?

When a feature is selected in 2.27.0 in Safari, I'm unable to move the keyboard focus away from fields like Operator in the sidebar. This makes it impossible to edit tags or use hotkeys. Firefox still works fine for me.

### What version numbers does this issue effect?

2.27.0

### Which browsers are you seeing this problem on?

Safari

**Comment by coniffer** [comment]
I also have this issue with iD 2.27.0 on Safari

**Comment by coniffer** [comment]
I have also noticed this issue on Firefox, Google Chrome, and DuckDuckGo browser

**Comment by kjonosm** [comment]
Has this been fixed in 2.27.1 ?

**Comment by quincylvania** [comment]
Yes, this is probably #9848.
